### ISSUE TITLE
Quick pick description text has low contrast

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#88752]
**Environment Details:​**
VSCode Version: 1.42.0-insider (System setup)
OS Version: Win10

**Steps to Reproduce:**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4. Navigate till "+" button beside the  "VS ONLINE" control and click on it .
5. Select any existing environment and then again click on the "+" button .
6. Open AI4D tool and select color contrast.
7. Check the color contrast for the text present in the screen.

**Actual:**
For the texts, The color contrast ratio is less than 4.5.

**Expected:​**
For the texts, The color contrast ratio should be >= 4.5.

**Same issue Observed for the below scenario:**
**1.VS ONLINE-Select Plan**
1.Open VS Code insider 
2.Navigate till "Extension" button and search for the "Visual Studio Online"
3.Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4.Navigate till "Select Plan" button beside the  "VS ONLINE" control and click on it.

**​User Impact:​**
Low vision user does not understand control if it is not  visible clearly.

![image](https://user-images.githubusercontent.com/55909580/72521937-7fd83080-3882-11ea-86ac-fc96640cfcd9.png)

**Comment by isidorn** [comment]
Seems like we will have to increase the contrast between name and description. Forwarding to @chrmarti so he considers this feedback

**Comment by misolori** [comment]
We'll need to drop the opacity from `0.7` to `0.9` to make this pass.

**Comment by isidorn** [comment]
@chrmarti @misolori Can we tackle this? Feels like low hanging fruit?

**Comment by misolori** [comment]
I updated the description text opacity on the light theme so it passes. Though I will say that this had to be set to `0.95` which makes it hard to see the difference. Part of the problem is that on light theme the quick pick bg color has some gray in it which makes it hard to find additional shades that pass.

<img width="50%" src="https://user-images.githubusercontent.com/35271042/84200542-d7f31780-aa5b-11ea-9824-605f4c8f941f.png">

<img width="50%" src="https://user-images.githubusercontent.com/35271042/84200565-e0e3e900-aa5b-11ea-8e09-e0725a04c634.png">

**Comment by isidorn** [comment]
Thanks! Yeah I do agree it is not ideal now since the description is very much similar to the main label. Idea for the future: make the color of the description customisable so theme authors can change it.
